lib/cfg.js

```diff
@@ -93,38 +92,30 @@ const gccX86 = {
     isJmpInstruction: x => x.trim()[0] === 'j',
 };
 
-const clangX86 = {
+const clang = {
     filterData: asmArr => {
         const jmpLabelRegex = /\.LBB\d+_\d+:/;
         const isCode = x => x && x.text && (x.source !== null || jmpLabelRegex.test(x.text) || isFunctionName(x));
 
-        const removeComments = x => {
-            const pos = x.text.indexOf('#');
-            if (pos !== -1)
-                x.text = utils.trimRight(x.text.substring(0, pos));
-            return x;
-        };
-
         return _.chain(filterTextSection(asmArr))
             .map(_.clone)
             .filter(isCode)
-            .map(removeComments)
             .value();
     },
     isFunctionEnd: x => x[0] !== ' ' && x[0] !== '.' && x.includes(':'),
 
     isBasicBlockEnd: (inst, prevInst) => inst[0] === '.' || prevInst.includes(' ret'),
 
+    isJmpInstruction: x => x.trim()[0] === 'j' || x.match(/\b(b)\.*(eq|ne|cs|hs|cc|lo|hi|ls|ge|lt|gt|le)*\b/),
```

Comment:
few questions on the regex:
`\.*` <-- are we actually looking for zero or more period characters here? if so, can you provide an example line this is meant to match?
`(eq|ne|cs|hs|cc|lo|hi|ls|ge|lt|gt|le)*` <- presumably we only want 0 or 1 of this? if so, we should replace `*` with `?`

Reply:
Regarding the zero or more periods, this was intentional. Arm64 uses a period between `b` and the condition code while previous versions omit it (ex: `beq`).